pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "cumulus-publish-cnm"
+version = "0.2.0"
+description = "Reads in list of CNM's and publishes to SNS topic"
+authors = ["Hong-Kit Randy Yeung <hong-kit.r.yeung@jpl.nasa.gov>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+cumulus-process = "^1.0.0"
+
+[tool.poetry.dev-dependencies]
+pytest = "^7.0.1"
+boto3 = "^1.18.40"
```

Comment:
even it's dev dependency, cumulus-process has dependency on boto3 an it will be automatically loaded

Reply:
ah it's only pulled in during development build and not the other build phases; it's used for when testing
When you run the build, the zip file would only contain boto3 which would be pulled in from cumulus-process
https://python-poetry.org/docs/cli/#options-3
boto3 was added in with `-D` flag